app/scripts/modules/core/src/search/infrastructure/RecentlyViewedItems.tsx

```diff
@@ -0,0 +1,76 @@
+import { IPromise } from 'angular';
+import * as React from 'react';
+import { BindAll } from 'lodash-decorators';
+import { Observable } from 'rxjs/Observable';
+import { Subject } from 'rxjs/Subject';
+
+import { IRecentHistoryEntry } from 'core/history';
+import { ReactInjector } from 'core/reactShims';
+
+import { ISearchResult, ISearchResultPodData, SearchResultPods } from './SearchResultPods';
+
+export interface IRecentlyViewedItemsState {
+  recentItems: ISearchResultPodData[];
+}
+
+@BindAll()
+export class RecentlyViewedItems extends React.Component<{}, IRecentlyViewedItemsState> {
```

Comment:
this component is responsible for fetching the data and passing it to `SearchResultPods`